include/libobmcsession/obmcsession.hpp

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
It seems to me this header is excess at all.
The C++ app could use the C-style API.

Reply:
In some cases C++ services use sdbusplus::bus::bus and it is too difficult to retrieve the sd_bus-C-style handle to pass into the C-style API functions of the current library. So, I prefer to define both API where it required,

Reply:
Yep, `sdbusplus::bus` has `get()` method, but unfortunately, it's protected.